### PR TITLE
8.15.0

### PR DESCRIPTION
**Pull request by bimac**

### CODE REVIEW

File: iblrig/gui/wizard.py [hunk 1]
```diff
@@ -454,6 +457,9 @@ def _on_menu_training_level_v7(self) -> None:
             self.uiGroupTaskParameters.findChild(QtWidgets.QWidget, '--adaptive_reward').setValue(reward_amount)
             self.uiGroupTaskParameters.findChild(QtWidgets.QWidget, '--training_phase').setValue(training_phase)
 
+    def _on_calibrate_frame2ttl(self) -> None:
+        Frame2TTLCalibrationDialog(self)
```
Comment (oliche) [inline iblrig/gui/wizard.py:461]: Can anyone run this CalibrationDialog without the GUI ? 
It seems quite useful to have a module that does just that if someone doesn't use the GUI.

If this is written in a modular manner, this is the kind of things that could eventually go in Josh repo and serve the community

**Review by oliche — CHANGES_REQUESTED** [review]
Olà thanks

1. the GUI tests have reverted to using the local parameters if they exist, which makes the test dependent on local state. This needs fixing before release
2. is there a smart way to mock the Frame2ttl object to test the calibration ? 
3. the calibration should be a callable module outside of the wizard. This piece may be used independently of iblrigv8

Please fix 1. before release, 2 and 3 could be marked as technical debt/issues for a future release.

File: iblrig/gui/wizard.py (hunk 1 above)
Comment (bimac) [reply iblrig/gui/wizard.py:461]: I kept the QT dependent part of the calibration within `wizard.py` for now, as it is quite specific to iblrig (size of calibration target etc). The calibration routine itself (`Frame2TTL.calibrate()`) works fine outside of iblrig - the user just has to supply a dark/light background to calibrate for

Comment (bimac) [reply iblrig/gui/wizard.py:461]: Regarding the GUI tests: I didn't change anything about the tests, other than auto-formatting. Could you point me to a line-number?

Working on the mocking now ...

**Pull request merged**